c2rust-refactor/src/rewrite/strategy/print.rs

```diff
@@ -144,7 +144,15 @@ impl PrintParse for Block {
 
 impl PrintParse for Arg {
     fn to_string(&self) -> String {
-        pprust::arg_to_string(self)
```

Comment:
Is this something that will be added to nightly soon?

Reply:
CC @rinon - Think you added this?

Reply:
It's already in nightly, but after the change that made the pretty printer impossible for us to use.